CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 # Change Log
 
+## Dangerzone 0.4.1 (unreleased)
+
+- Platform support: Drop Fedora 35, since it's end-of-life ([issue #308](https://github.com/freedomofpress/dangerzone/issues/308))
+- Feature: Introduce PySide6 / Qt6 support on Windows, MacOS, and Linux (dev-only) ([issue #219](https://github.com/freedomofpress/dangerzone/issues/219))
+- Feature: Adjust conversion timeouts based on the document's pages/size, and
+  allow users to disable them ([issue #327](https://github.com/freedomofpress/dangerzone/issues/327))
```

Comment:
Also, is it worth mentioning that these issues were also fixed with this?

- https://github.com/freedomofpress/dangerzone/issues/314
- https://github.com/freedomofpress/dangerzone/issues/306

Reply:
How about:

> allow users to disable them with `--disable-timeouts` (available when you run the Dangerzone CLI/GUI from the terminal)

As for the other two issues, I'm not sure if users reading the changelog will gain much from them, since the umbrella issue already covers them. But I can add them if we aim for completeness.

Reply:
I don't think the "GUI from the terminal" part should be mentioned. We don't advertise that anywhere and it may confuse users. Maybe just `(available when you run Dangerzone through the terminal)`?